edb/tools/wipe.py

```diff
@@ -0,0 +1,192 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2016-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from __future__ import annotations
+from typing import *  # NoQA
+
+import asyncio
+import json
+import pathlib
+import sys
+
+import asyncpg
+import click
+
+from edb.common import topological
+from edb.tools.edb import edbcommands
+
+from edb.server import cluster as edbcluster
+from edb.server import compiler
+from edb.server import defines as edbdef
+from edb.server import pgcluster
+from edb.server import pgconnparams
+
+from edb.pgsql.common import quote_ident as qi
+
+
+class AbsPath(click.Path):
+    name = 'path'
+
+    def convert(self, value, param, ctx):
+        return pathlib.Path(super().convert(value, param, ctx)).absolute()
+
+
+@edbcommands.command('wipe')
+@click.option(
+    '--postgres-dsn',
+    type=str,
+    help='DSN of the remote Postgres instance to wipe EdgeDB from')
+@click.option(
+    '-D',
+    '--data-dir',
+    type=AbsPath(),
+    help='database cluster directory')
+@click.option(
+    '-y',
+    'yes',
+    is_flag=True,
+    help='assume Yes response to all questions')
+def wipe(*, postgres_dsn, data_dir, yes):
```

Comment:
Would be great to also add `--dry` to do a dry run.

Reply:
Done.